tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -87,6 +87,16 @@ internal static partial class ConfigurationKeys
         /// </summary>
         internal const string AppSecWafTimeout = "DD_APPSEC_WAF_TIMEOUT";
 
+        /// <summary>
+        /// The regex that will be used to obfuscate possible senative data in keys that are highlighted WAF as potentially malicious
+        /// </summary>
+        internal const string AppSecObfuscationParameterKeyRegex = "DD_APPSEC_OBFUSCATION_PARAMETER_KEY_REGEXP";
+
+        /// <summary>
+        /// The regex that will be used to obfuscate possible senative data in values that are highlighted WAF as potentially malicious
+        /// </summary>
+        internal const string AppSecObfuscationParameterValueRegex = "DD_APPSEC_OBFUSCATION_PARAMETER_VALUE_REGEXP";
```

Comment:
Not for this PR, but might be nice to move all the `DD_APPSEC_*` keys to their own `ConfigurationKeys.AppSec.cs` file?

Reply:
Yes, good idea. I'll make a separate PR with that.